### PR TITLE
Audit event edits

### PR DESCRIPTION
**Pull request by mikejritter**
Updates based on comments from 

* https://github.com/ucsdlib/otm-specs/issues/19
* https://github.com/ucsdlib/otm-specs/issues/31

### CODE REVIEW

File: audit-appendix.html [hunk 1]
```diff
@@ -98,6 +102,8 @@ <h3>Ingest</h3>
       <h3>Replication</h3>
       <p>The Replication event represents the distribution of a filegroup or a file to a single location within a DDP.</p>
 
+      <p>It is up to the discretion of the DDP if it wants to provide a Replication event for each copy of the filegroup made or
```
Comment (mikejritter) [inline audit-appendix.html]: Would like to get some feedback on this. I wanted to clarify that DDPs operate differently, and that Replication events can be either for all copies or for the primary copy. Generally we do for all so we could also modify this to just say a Replication event should exist for each copy.

**Comment by mikejritter** [comment]
There was an additional comment from Andrew on the Ingest event:
```
Curious if it's worth noting the DDP's internal identifier?
```

I was on the fence about this, but am not opposed to adding it into the appendix. Just wanted to discuss it before putting it in.

File: audit-appendix.html [hunk 2]
```diff
@@ -96,7 +100,8 @@ <h3>Ingest</h3>
 
     <section>
       <h3>Replication</h3>
-      <p>The Replication event represents the distribution of a filegroup or a file to a single location within a DDP.</p>
+      <p>The Replication event represents the distribution of a filegroup or a file to a single location within a DDP. This should
+      occur for each copy of the filegroup regardless of whether or not it is the primary copy or not.</p>
```
Comment (bbranan) [inline audit-appendix.html]: Could probably leave off the final "or not" here. The current "whether or not it is the primary copy or not" sounds odd.

Comment (mikejritter) [reply audit-appendix.html]: Agreed. Updated to fix the wording.

**Pull request merged**